lib/dl_connector_chyt/dl_connector_chyt/core/adapters.py

```diff
@@ -8,7 +8,7 @@
     Type,
 )
 
-from aiochclient.http_clients import aiohttp
+import aiohttp
```

Comment:
Was there a particular reason to use it here in the first place? Or it was a bad import all along

Reply:
I think it was just a mistake (or some kind of PyCharm's incorrect auto-replacement).
There was usual aiohttp module inside.